assets/source/sass/layout/_main-content.scss

```diff
@@ -41,6 +41,10 @@
         color: #777;
     }
 
+    .comment-form-comment {
+        background-color: #fff; // Only affects upphandlingsbloggen since comments are disabled on main site.
```

Comment:
We should probably be careful with stuff like this so that it doesn't leak across subsites when it's meant specifically for one of them.